docs/contact-list.md

```diff
@@ -0,0 +1,28 @@
+# Security Contact List
+
+Beyond the initial fix team, a mailing list is setup to include additional
+organizations that can benefit from early CVE access. This mailing list
+is created per repository
+
+## OpenTelemetry Collector Distro Security Contacts List
+
+Currently on the opentelemetry-collector-distros list are representatives from:
+
+*
+*
+
+## Membership Criteria
+
+To be eligible for (opentelemetry-)distros list membership, your distro should:
+
+* Be an actively maintained Open Source OpenTelemetry distribution
+* Have a userbase not limited to your own organization
+* Have a publicly verifiable track record, dating back at least 1 year and
+  continuing to present day, of fixing security issues and releasing the 
+  fixes within 10 days of the issues being made public
```

Comment:
Would "10 days" be too aggressive? I think depending on the actual severity score, it might be fine to bundle low severity fixes in the next release train, which is normally more than 10 days.

Reply:
i don't have a strong preference here, this was the timeline suggested by the linux distributions document